drf_extra_fields/geo_fields.py

```diff
@@ -76,3 +77,77 @@ def to_representation(self, value):
             value['latitude'] = smart_str(value.pop('latitude'))
 
         return value
+
+
+
+class PolygonField(serializers.Field):
+    """
+    A field for handling GeoDjango PolyGone fields as a array format.
+    Expected input format:
+        {
+            [
+                [
+                    51.778564453125,
+                    35.59925232772949
+                ],
+                [
+                    50.1470947265625,
+                    34.80929324176267
+                ],
+                [
+                    52.6080322265625,
+                    34.492975402501536
+                ],
+                [
+                    51.778564453125,
+                    35.59925232772949
+                ]
+            ]
+        }
```

Comment:
This doesn't support polygon with inner rings. 

How about we use the following format with the key `coordinates`?

**Polygon with single ring** (The way you implemented.)
```
coordinates : [
     [ [ 2 , 2 ] , [ 3 , 3 ] , [ 4 , 2 ] , [ 2 , 2 ] ]
  ]
```

**Polygon with multiple rings** (The one that is not supported at the moment.)
```
coordinates : [
     [ [ 0 , 0 ] , [ 3 , 6 ] , [ 6 , 1 ] , [ 0 , 0 ] ],
     [ [ 2 , 2 ] , [ 3 , 3 ] , [ 4 , 2 ] , [ 2 , 2 ] ]
  ]
```

From geojson specification
> The "coordinates" member must be an array of LinearRing coordinate arrays. For Polygons with multiple rings, the first must be the exterior ring and any others must be interior rings or holes. 

Please see [django documentation](https://docs.djangoproject.com/en/3.1/ref/contrib/gis/geos/#polygon) for usage.